src/igr-tests/igr-runner.cc

```diff
@@ -14,12 +14,12 @@ int main(int argc, char **argv)
 {
     const char * const test_dirs[] = { "basic", "environ", "environ2", "ps-environ", "chain-to", "force-stop",
             "restart", "check-basic", "check-cycle", "check-cycle2", "check-lint", "reload1", "reload2",
-            "no-command-error", "add-rm-dep", "var-subst", "svc-start-fail", "dep-not-found", "pseudo-cycle" };
+            /* "no-command-error", "add-rm-dep", "var-subst", "svc-start-fail", "dep-not-found", "pseudo-cycle" */};
     constexpr int num_tests = sizeof(test_dirs) / sizeof(test_dirs[0]);
 
-    int passed = 0;
-    int skipped = 0;
-    int failed = 0;
+    unsigned int passed = 0;
+    unsigned int skipped = 0;
+    unsigned int failed = 0;
```

Comment:
@davmac314 I think it's must be `unsigned short int` for better memory management. No?

Reply:
Changing these to `short` would literally save 6 bytes, while running the integration tests only. I really have better things to worry about!